app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -21,13 +21,12 @@ def index
   end
 
   def get_meeting_info
```

Comment:
Please remove these unrelated changes; there's no reason for this PR to touch API methods other than create and join.